package.json

```diff
@@ -36,6 +36,7 @@
     "eslint-loader": "^1.9.0",
     "extract-text-webpack-plugin": "^3.0.0",
     "node-sass": "^4.5.3",
+    "nunjucks": "^3.0.1",
```

Comment:
I've got some views on nunjucks, happy to talk about them.

Some of my thoughts here:
https://github.com/wellcometrust/wellcomecollection.org/pull/740